tests/unit/components/LinkList.test.js

```diff
@@ -1,4 +1,4 @@
-import { shallowMount } from '@vue/test-utils';
+import testHelpers from '@@/unit/test-helpers.js';
```

Comment:
I've updated all tests to use helper functions for mounting, so any set up can be done in one place.